fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1111,6 +1147,19 @@ ACTOR Future<Void> applyMetadataToCommittedTransactions(CommitBatchContext* self
 	return Void();
 }
 
+void writeMutation(CommitBatchContext* self, int64_t tenantId, const MutationRef& mutation) {
+	// TODO: encrypt system keys after support encrypting txnStateStore.
+	if (!SERVER_KNOBS->ENABLE_TLOG_ENCRYPTION) {
+		self->toCommit.writeTypedMessage(mutation);
+		return;
+	}
+	Arena arena;
+	static_assert(TenantInfo::INVALID_TENANT == ENCRYPT_INVALID_DOMAIN_ID);
+	EncryptCipherDomainId domainId =
```

Comment:
Per offline discussion it can be from a raw access transaction, where tenant info is missing. Updated with a `TEST` macro and a TODO to address raw access later (where we need to figure out tenant id from tenant prefix and get tenant name from a tenant prefix index maintained either by CP itself or by EKP).